src/ipc-types.d.ts

```diff
@@ -21,7 +21,10 @@ interface StoppedSiteDetails {
 	phpVersion: string;
 	wpVersion?: string;
 	customDomain?: string;
+	enableSSL?: boolean;
```

Comment:
```suggestion
	enableHttps?: boolean;
```
I know "SSL" is often used interchangeably with "HTTPS", but I don't believe we actually use any SSL here. Instead of changing to `enableTLS`, `enableHttps` seems to convey the meaning more effectively.

Reply:
If you look at the commit history, I went back and forth :P. I'm not strongly opinionated